src/main/interface/cli.c

```diff
@@ -4536,7 +4547,7 @@ const clicmd_t cmdTable[] = {
     CLI_COMMAND_DEF("flash_write", NULL, "<address> <message>", cliFlashWrite),
 #endif
 #endif
-#ifdef USE_RX_CC2500_BIND
+#ifdef USE_RX_SPI
```

Comment:
Not your fault, but you're the last to touch it: :stuck_out_tongue_closed_eyes: 
`CLI_COMMAND_DEF` need to be ordered alphabetically, or else tab completion will break. I am also thinking if we should rename `bind` to `spi_bind` or `bind_spi` to make it clearer what it is for.

Reply:
Agreed, it should be moved, no worries.
Renamed to bind_rx_spi, and moved up.